CNAME

```diff
@@ -1 +0,0 @@
-read.wiki.org
```

Comment:
Ah, I just realized why I thought this was going to be a problem, but it isn't. My issue was merging to `gh-pages` with the commit from this PR that actually removes `CNAME`. We'll just need to restore `CNAME` on our branch, and then future merges will carry through seamlessly.

Sorry for the noise @paul90, happy with the change now!